### DISCUSSION TITLE
What does the "Opacity Step" setting do?

### DISCUSSION THREAD

**Discussion by superscratch444tre** [ScratchAddons/ScratchAddons#1341] [Help] [answered]
Hi! I do not understand what does the "Opacity Step" setting in the onion skinning addon do.

It looks the same if I set it to 90 or to 10.

**Comment by cobaltt7** [comment]
How see-through the onion skin is.
________________________________

**Comment by mxmou** [comment, marked as answer]
It only has effect if "Next costumes" or "Previous costumes" is set to more than 1. It's the difference in opacity of the previous/next costumes.
